pages/api/class/index.ts

```diff
@@ -15,7 +16,11 @@ export default async function handle(
 ): Promise<void> {
     switch (req.method) {
         case "GET": {
-            const classes = await getClasses();
+            const { id } = req.query;
+
+            const classes = id
+                ? await getClassInfoWithProgramId(id as string)
```

Comment:
Let's convert `programId` to a number and use it as type number into this function. 

See below for example of converting to a number
https://github.com/uwblueprint/social-diversity-for-children/blob/862fabc0e0dd0709180aea47a5ddb30ae6e187d7/pages/api/enroll/child.ts#L43-L63